data/human/news.txt

```diff
@@ -1695,6 +1853,13 @@ news "pirate"
 			"Living on the edge."
 			"You can't win the game if you're afraid to lose it."
 			"Good things come to those who wait, and then I rob those from them very quickly."
+			"Two heads are better than one, since you can put a spiked head on each side of the road if you have two."
+			"It takes two to make a bargain. Just kill the first fool that says no, and the second one will quickly agree to my demands."
+			"I fought the law, and I won."
+			"Sorry, need to take a call. 'Hi, this is Joe from Syndicate support. You seem to have a problem with your flight computer.'"
```

Comment:
Can the single quotes be double quotes here? Sorry, I'm not sure if backticks are supported in this place.

Reply:
They are.